### PR TITLE
how to building #283

### PR DESCRIPTION
**Pull request by mbenguig**
Could Brian confirm this PR?

### CODE REVIEW

**Comment by mbenguig** [comment]
Build section updated, thanks to the Laurent support mail.

File: README.md [hunk 1]
```diff
@@ -1,14 +1,17 @@
-README
+# ProActive Workflows & Scheduling
```
Comment (marcocast) [inline README.md]: I think we changed the name to "ProActive Scheduling & Orchestration"

File: README.md [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
+[![Build Status](http://jenkins.activeeon.com/job/scheduling-portal/badge/icon)](http://jenkins.activeeon.com/job/scheduling-portal/)
 
-### 1. Description #############################################################
 
-This project is a Web front-end for the ProActive Scheduler and Resource
-Manager REST API. It does not require the REST API to be built, but will not be
-able to properly run without it.
+## Description
+
+This project is a Web front-end for the ProActive Scheduler and Resource Manager
```
Comment (marcocast) [inline README.md]: ProActive Scheduling & Orchestration instead of ProActive Scheduler

File: README.md [hunk 3]
```diff
@@ -17,79 +20,92 @@ This project is Open Source and distributed under the terms of the GNU AGPLv3,
...
+  - Scheduler and RM portals: Checkout the project scheduling-portal
+  (https://github.com/ow2-proactive/scheduling-portal). Execute 'gradle build'.
+  The command will produce two wars that you need to symlink to SCHEDULING_HOME/dist/war:
 
-To build the project, use the /compile/build[.bat] script:
-  - Unix:    $ cd compile
-             $ ./build wars
-  - Windows: > cd compile
-             > build.bat wars
+    `$> ln -s SCHEDULING_PORTAL_HOME/rm-portal/build/libs/rm-portal-6.3.0-SNAPSHOT.war SCHEDULING_HOME/dist/war/rm.war`
```
Comment (marcocast) [inline README.md]: Do we need to put the snapshot version?

File: README.md (hunk 1 above)
Comment (mbenguig) [inline README.md]: I was confirmed  "ProActive Scheduling & Orchestration" is only the name of one of our partals, part of the "ProActive Workflows & Scheduling" solution.

File: README.md (hunk 3 above)
Comment (mbenguig) [inline README.md]: ...-SNAPSHOT...  removed (x2)

**Pull request merged**